src/framework/enhancement/js/ContextAwareness.js

```diff
@@ -204,15 +204,18 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         }
     });
 
-    // Context awareness for the platform (operating system)
+    // Context awareness for the reported browser platform name (operating system)
 
-    fluid.contextAware.getPlatformName = function () {
-        return navigator.platform ? navigator.platform : undefined;
+    fluid.contextAware.getBrowserPlatformName = function () {
+        // Gets platform name in browser contexts
+        if (typeof(navigator) !== "undefined" && navigator) {
+            return navigator.platform ? navigator.platform : undefined;
```

Comment:
Is it necessary to do the conditional check on `navigator.platform`? Can you just do `return navigator.platform` or could it have a value that is falsey and not undefined that you want to catch?